config/settings.yml

```diff
@@ -854,7 +854,7 @@ maintenance:
     carma: P6XLE0T
     appeals: P9S4RFU
     arcgis: P45YBFA
-    coe: PXXXXXX
+    coe: PSY4HU1
```

Comment:
https://dsva.pagerduty.com/service-directory/PSY4HU1

I'm not sure how this got here in the first place. The git blame points to PR https://github.com/department-of-veterans-affairs/vets-api/pull/11678, which makes sense until you see Eric reverted the addition in https://github.com/department-of-veterans-affairs/vets-api/commit/711a0d55da0a67c4a55b81b2afeb8412c0c6ead9 😕 deleting it for good here!

This value was being overwritten in prod and staging values.yaml, but in sandbox and dev, this `PXXXXXX` was getting passed to pagerduty. I found it in dev:
```
irb(main):005> Settings.maintenance.services
=> #<Config::Options carma="P6XLE0T", appeals="P9S4RFU", arcgis="PAWZGRJ", coe="PXXXXXX", ...>
irb(main):006> 
```

Reply:
Oh yes! Thanks for pointing that out